cmd/markdown/atmos_terraform_plan_usage.md

```diff
@@ -0,0 +1,7 @@
+Examples:
+
+  – Execute a terraform plan
+
+```
+    $ atmos terraform plan <component-name> -s <stack-name>
+```
```

Comment:
```suggestion
  – Execute a terraform plan

    ```
        $ atmos terraform plan <component-name> -s <stack-name>
    ```
```

Reply:
I can do that but my observation showed it does not make any difference for the markdown.